lib/dmv.rb

```diff
@@ -9,7 +9,7 @@ def add_facility(facility)
   end
 
   def facilities_offering_service(service)
-    @facilities.find do |facility|
+    @facilities.find_all do |facility|
       facility.services.include?(service)
     end
```

Comment:
Great use of `find_all`!

Reply:
Thanks!  I might have had Kat hold my hand into that one ;)